lib/cocoapods-core/source/manager.rb

```diff
@@ -389,6 +394,9 @@ def name_for_url(url)
           base = 'master'
         when %r{github.com[:/]+(.+)/(.+)}
           base = Regexp.last_match[1]
+        when %r{^ssh:\/\/\S+@(\S+):{1}(.+)$}
```

Comment:
why isn't this being handed by the case right below? can we merge the two into a single regexp?

Reply:
I try merge it into a single regexp, but failure.